src/HttpCall/HttpCallListener.php

```diff
@@ -51,9 +51,11 @@ public function afterStep(AfterStepTested $event)
         // For now to avoid modification on MinkContext
         // We add fallback on Mink
         try {
-            $this->httpCallResultPool->store(
-                new HttpCallResult($this->mink->getSession()->getPage()->getContent())
-            );
+            if ($this->mink->getSession()->isStarted()) {
```

Comment:
Note that this broke our test suite

Reply:
Same, and 2 other people experienced this issue too here : https://github.com/soyuka/contexts/commit/8215019f4f75d3e184f70b075e5a352b6551f0bc#r109969509